src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php

```diff
@@ -202,6 +205,43 @@ private function addClassesSection(ArrayNodeDefinition $node)
                                             ->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('translation')
+                                        ->isRequired()
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->variableNode('option_value')->end()
+                                            ->arrayNode('classes')
+                                                ->isRequired()
+                                                ->addDefaultsIfNotSet()
+                                                ->children()
+                                                    ->scalarNode('model')->defaultValue(OptionValueTranslation::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('interface')->defaultValue(OptionValueTranslationInterface::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('repository')->cannotBeEmpty()->end()
+                                                    ->scalarNode('factory')->defaultValue(Factory::class)->end()
```

Comment:
ok, thanks. I fixed in this [commit](https://github.com/vikey89/Sylius/commit/fc55513bb4608d721ae9aefac8f82724eb908c3d.).